client/packages/inventory/src/Stocktake/DetailView/modal/StocktakeLineEdit/StocktakeLineEditTables.tsx

```diff
@@ -146,13 +150,21 @@ const getInventoryAdjustmentReasonInputColumn = (
   };
 };
 
-export const BatchTable: FC<StocktakeLineEditTableProps> = ({
-  batches,
-  update,
-  isDisabled = false,
-}) => {
+// If this is not extracted to it's own component and used directly in Cell:
+// cell will be re rendered anytime rowData changes, which causes it to loose focus
+// if number of packs is changed and tab is pressed (in quick succession)
+const PackUnitEntryCell = PackVariantEntryCell<DraftStocktakeLine>({
+  getItemId: r => r.item.id,
+  getUnitName: r => r.item.unitName || null,
+  getIsDisabled: r => !r.isNewLine,
```

Comment:
Have to pass it down to the components since the cell isDisabled doesn't work for this